WordPress/Classes/Models/Blog/Blog.m

```diff
@@ -569,9 +572,13 @@ - (BOOL)supports:(BlogFeature)feature
         case BlogFeatureStats:
             return [self supportsRestApi] && [self isViewingStatsAllowed];
         case BlogFeatureStockPhotos:
-            return [self supportsRestApi] && [JetpackFeaturesRemovalCoordinator jetpackFeaturesEnabled];
+            // FIXME: See https://github.com/wordpress-mobile/WordPress-iOS/pull/24332
+//            return [self supportsRestApi] && [JetpackFeaturesRemovalCoordinator jetpackFeaturesEnabled];
+            return [self supportsRestApi];
         case BlogFeatureTenor:
-            return [JetpackFeaturesRemovalCoordinator jetpackFeaturesEnabled];
+            // FIXME: See https://github.com/wordpress-mobile/WordPress-iOS/pull/24332
+//            return [JetpackFeaturesRemovalCoordinator jetpackFeaturesEnabled];
+            return NO;
```

Comment:
https://github.com/wordpress-mobile/WordPress-iOS/pull/24393 should address this.